### ISSUE TITLE
Meta Dorms Cabin lockers are wrong

### ISSUE THREAD

**Issue by Googles-Hands** [tgstation/tgstation#63105]
## Reproduction:
The wooden lockers in the six dormitory cabins on MetaStation which each contain leather satchels and some other random gear have been replaced with the standard metal personal closet's

**Comment by timothymtorres** [comment]
This is intended.  Maps are getting random spawners for objects and structures added to them.  Right now dorms have a 50% chance to spawn either a wooden locker or the metal personal closet.

**Comment by LemonInTheDark** [comment]
@timothymtorres I'm minorly cheesed, since this wasn't a change you appear to have documented anywhere. From what I can tell it was done in https://github.com/tgstation/tgstation/pull/62929, but I only know that because of what you've said here (dmm blames can be a real headache to deal with). You didn't mention it in the pr desc at all.

Not horribly cheesed since it was mentioned in the tram pr and @Maurukas was good with that, but still.

**Comment by timothymtorres** [comment]
The cosmetic random spawners PRs for Meta and Tram were the first time I have made large mapping changes.  I brought this up in discord to the maptainers a few weeks ago on how best I should implement these changes and there was talk about a mapping freeze so I wouldn't have to fight repeating merge conflicts.  I decided to break these changes into separate PRs for each map so I wouldn't need to do a mapping freeze.  (95% of the changes are identical across all maps)

> @timothymtorres I'm minorly cheesed, since this wasn't a change you appear to have documented anywhere.

The cabin lockers in dorms for Meta was something I missed with my documentation apparently.

> The wooden lockers in the six dormitory cabins on MetaStation which each contain leather satchels and some other random gear have been replaced with the standard metal personal closet's

The gear is cosmetic.  The code shows the following:

https://github.com/tgstation/tgstation/blob/15629905cd896a13e326abc747b7a707ef0944fb/code/game/objects/structures/crates_lockers/closets/secure/personal.dm#L7-L15

https://github.com/tgstation/tgstation/blob/15629905cd896a13e326abc747b7a707ef0944fb/code/game/objects/structures/crates_lockers/closets/secure/personal.dm#L33-L36
